### DISCUSSION TITLE
Retain Formula after execute Fetch Tables

### DISCUSSION THREAD

**Discussion by Kakovisk** [Budibase/budibase#9733] [Ideas]
Hi everyone,

When I create a Formula in a Table  its delete when I execute Fetch tables.

My suggestion is the possibility of retain this formula after to execute Fetch tables.

Tks.

**Comment by melohagan** [comment]
Hey @Kakovisk 

I agree this would be good! A workaround is to fetch specific tables so that your Formula columns are not overwritten unless you are updating that specific table: https://docs.budibase.com/docs/sql-datasource#fetching-specific-tables

**Comment by melohagan** [comment]
This one has been addressed! https://github.com/Budibase/budibase/pull/9923
